examples/wandb_sweeps/train_wandb_sweep_agent.py

```diff
@@ -0,0 +1,178 @@
+import os
+import traceback
+import sys
+import math
+from typing import Tuple
+import functools
+
+import wandb
+import pytorch_lightning as pl
+import argparse
+
+from yoyodyne import (
+    collators, dataconfig, defaults, models, schedulers, train, predict, util
+)
+
+
+def get_args():
```

Comment:
The part of this that shadows `train.py` should probably just be moved into a function there, then we can import it here and add additional arguments here as needed. WYDT?

Reply:
Yeah that makes sense.